.github/workflows/release-samples.yml

```diff
@@ -24,6 +24,9 @@ jobs:
         run: |
           python -m pip install --upgrade pip
 
+      - name: Set current date as env variable
+        run: echo "NOW=$(date +'%Y-%m-%dT%H:%M:%S')" >> $DATETIME
```

Comment:
I think this should be `>> $GITHUB_ENV`.

Reply:
Ah yeah, got the syntax mixed up